pkgs/development/libraries/nv-codec-headers/make-nv-codec-headers.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenvNoCC
+, fetchgit
+}:
+
+{ pname ? "nv-codec-headers"
+, version
+, hash
+}:
+
+stdenvNoCC.mkDerivation {
+  inherit pname version;
+
+  src = fetchgit {
+    url = "https://git.videolan.org/git/ffmpeg/nv-codec-headers.git";
+    rev = "n${version}";
+    inherit hash;
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = {
+    description = "FFmpeg version of headers for NVENC";
+    homepage = "https://ffmpeg.org/";
```

```suggestion
    homepage = "https://ffmpeg.org/";
    downloagPage = "https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git";
```
because when you paste the git url in a browser, you just get a 404